README.md

```diff
@@ -50,6 +51,17 @@ choco install sass
 That'll give you a `sass` executable on your command line that will run Dart
 Sass.
 
+### From Scoop (Windows)
```

Comment:
Since this essentially duplicates the Chocolatey section, can you just merge the two into a single "From Chocolatey or Scoop (Windows)" section?